HackYourFuture-CPH/Guide-IT
# renamed create_questions_schema.js -> 20210427152901_create_questions…

…_table.js

# Description

I replaced old migration file with the  new created by using knex cli in PR 75

Fixes # (issue)
#28 

# How to test?
1. It should rename create_questions_schema.js -> 20210427152901_create_questions_table.js in migration folder.
2. please run:  npm run knex -- migrate:latest  to Run "latest" migrations and verify questions table in database.
3. run : npm run knex -- migrate:rollback to "Roll back" the latest migration.



# Checklist

- [x ] I have performed a self-review of my own code
- [ x] This PR is ready to be merged and not breaking any other functionality
